src/korean.js

```diff
@@ -93,6 +96,12 @@ class Korean {
     } // end of first else
   } // end of presentWord function
 
+  doPresentContinuous(word) {
+    let wordLength = word.length;
+    let conjugate = word.substring(0, wordLength-1);
+    return `${conjugate}고있어`;
```

Comment:
I'm going to approve your code, but you could reduce all to 1 line without compromising readability:
```
return `${word.substring(0, word.length)}고있어`
```